rdt-open-reader/src/test/java/io/ona/rdt/robolectric/widget/WidgetFactoryRobolectricTest.java

```diff
@@ -33,19 +33,27 @@ public void tearDown() {
     }
 
     public static RDTJsonFormActivity getRDTJsonFormActivity() throws JSONException {
+        return getRDTJsonFormActivity(new JSONObject());
+    }
+
+    public static RDTJsonFormActivity getRDTJsonFormActivity(JSONObject stepObject) throws JSONException {
```

Comment:
I don't think we'll need this extra function.

Reply:
I do this because this class `CovidRDTJsonFormUtilsTest` calling that `WidgetFactoryRobolectricTest.getRDTJsonFormActivity()` method inside a test